src/routes/transactions/mappers/transactions-history.mapper.ts

```diff
@@ -156,18 +155,19 @@ export class TransactionsHistoryMapper {
   ): TransactionDomainGroup[] {
     return Object.entries(
       groupBy(transactions, (transaction) => {
-        return this.getDayFromTransactionDate(
-          transaction,
-          timezoneOffset,
-        ).getTime();
+        return [
```

Comment:
Nit: I think returning an array here is not necessary for the grouping, right?
```javascript
groupBy(transactions, (transaction) =>
  this.getDayFromTransactionDate(transaction, timezoneOffset).getTime(),
)
```

Reply:
It is not. Good catch.

I was trying an alternative solution where the return would be a tuple that would include the UTC timestamp (to avoid indexing the first transaction in the followup `map`).

Reply:
Fixed in [918d1b3](https://github.com/safe-global/safe-client-gateway/pull/1006/commits/918d1b34c189ff1972da1d8a6589f97d08b79148)